### DISCUSSION TITLE
Fedora package building - RiscV

### DISCUSSION THREAD

**Discussion by hwspeedy** [hardinfo2/hardinfo2#19] [General]
@decathorpe

We are rebooting the nice old program hardinfo as a community edition hardinfo2.

Memory refresh:    https://hardinfo2.org

This program is ideal for SBC (Single board computers) like "RISC-V architecture" - It shows information and does benchmark testing.

The sysbench is not supported yet due to missing lua riscv support - so "hardinfo2" might be among the very few tests benches that can run on RISC-V right now.

**Do you know how to get this program into the test building pool for RISC-V:**
https://bodhi.fedoraproject.org/updates/?packages=hardinfo2
https://src.fedoraproject.org/rpms/hardinfo2

In advance thanx,

/hwspeedy

**Comment by decathorpe** [comment]
Hi! Not sure why I am tagged here directly. I'm not involved with the RISC-V bringup for Fedora. I'm not sure what the process is to get it built on http://riscv.rocks/koji/ , but I don't think you need to do anything.

**Comment by topazus** [comment]
Maybe you can turn to @rwmjones for some information about risc-v on Fedora.

**Comment by rwmjones** [comment]
Adding @davidlt

In general there is nothing you need to do here.  The new package was added to Fedora only 2 days ago.  In time it will be picked up and rebuilt in Fedora RISC-V.  You will eventually see it appear under here: http://fedora.riscv.rocks/koji/search?match=glob&type=package&terms=hardinfo2
